programs/survival/compass.sc

```diff
@@ -0,0 +1,604 @@
+// If you want your script to work with the persist flag, signal the relevant scarpet event
+// For example: signal_event('player_places_block', player, player, l('acacia_door', 16, null), 'mainhand', block(0, 0, 0));
+
+__config()->{
+    'scope' -> 'global',
+    'commands' -> {
+        'giveCompass' -> _() -> giveCompass(player()),
```

Comment:
Again here, try use snake case for functions etc.

Reply:
Ok, I've switched what I could find to snake case. From what I can tell it still seems to work

Imo, using snake case for the commands seems kinda pointless